### ISSUE TITLE
[BUG] PGFPlotsX LaTeXString rendering on second-time display

### ISSUE THREAD

**Issue by korsbo** [JuliaPlots/Plots.jl#2780]
## Details
LaTeXStrings stop working after the first display or save on the PGFPlotsX backend.

```
using Plots
pgfplotsx()
using LaTeXStrings
scatter(rand(1,2), label=L"x_y") ## pops up and looks fine
plot!() ## the latexstring is no longer rendered properly.
```

### Backends

This bug occurs on ( insert `x` below )

Backend      | yes | no  | untested
-------------|-----|-----|---------
gr (default) |     |     |x
pyplot       |     |     |x
plotly       |     |     |x
plotlyjs     |     |     |x
pgfplotsx    |x     |     |
inspectdr    |     |     |x

### Versions

Plots.jl version: 1.4.0
Backend  version (`]st -m`):
PGFPlotsX v 1.2.6
Output of `versioninfo()`:
```
Julia Version 1.4.1
Commit 381693d3df* (2020-04-14 17:20 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, skylake)
```

**Comment by BeastyBlacksmith** [comment]
can you try https://github.com/JuliaPlots/Plots.jl/pull/2781?

**Comment by BeastyBlacksmith** [comment]
Fixed by #2781
